cypress/e2e/tldraw/createTLdrawWithContent.feature

```diff
@@ -17,8 +17,9 @@ Feature: TLDraw - Test tldraw on board
         Given the multi-column board has a column with a card
 
         # teacher creates a whiteboard element in the card and draws on it
+        When I click on the three dot on the card
```

Comment:
something missing after this step?

Reply:
I added clicking on edit after this step, thanks.